mitodl/ocw-hugo-themes
# playwright base url override

# What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/1285

# Description (What does it do?)
This PR looks at a new env variable, `PLAYWRIGHT_BASE_URL`, for a base URL to run Playwright tests against. If this variable is set, none of the steps in `global-setup.ts` are performed. Playwright proceeds to run its tests targeting the site denoted by `PLAYWRIGHT_BASE_URL`.

# How can this be tested?
This PR will mostly be tested as part of https://github.com/mitodl/ocw-studio/pull/2018, although you should be able to still run `yarn test:e2e` locally and everything should still function. Github Actions end to end tests should also still pass.
